### PR TITLE
Add API change to docs

### PR DESCRIPTION
**Pull request by WeiZhang555**
Add API change description to docs due to `docker network inspect`
returns different data structure.

Signed-off-by: Zhang Wei zhangwei555@huawei.com

Relative comment is here https://github.com/docker/docker/pull/17615#issuecomment-156664525
and relative PR is #17615

ping @thaJeztah

### CODE REVIEW

File: docs/reference/api/docker_remote_api.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 [Docker Remote API v1.22](docker_remote_api_v1.22.md) documentation
 
 * `GET /containers/json` supports filter `isolation` on Windows.
+* `GET /networks/(name)` involves container `Name`.
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api.md]: Perhaps something like;

```Markdown
* `GET /networks/(name)` now returns a `Name` field for each container attached to the network.
```

**Comment by thaJeztah** [comment]
Thanks @WeiZhang555, I added one suggestion. Also can you update the man page with the same output; https://github.com/docker/docker/blob/master/man/docker-network-inspect.1.md

**Comment by WeiZhang555** [comment]
@thaJeztah Ah, you are right, I missed it. I have refreshed the docs. 

What's wrong with the documentation test? It fails but I don't understand the error report.

**Comment by thaJeztah** [comment]
@WeiZhang555 it's safe to ignore the documentation test for now, I think it's still being worked on :innocent:

**Comment by thaJeztah** [comment]
docs LGTM, thanks @WeiZhang555 !

ping @SvenDowideit @moxiegirl @jamtur01 for the merge :)

**Comment by jamtur01** [comment]
LGTM

**Pull request merged**